server/routes/handlers/helpers/viewWithRecallAndPerson.ts

```diff
@@ -61,5 +61,14 @@ export const viewWithRecallAndPerson =
         res.locals.recall.lastReleasePrison
       )
     }
+    if (recall.assessedByUserId) {
+      try {
+        const { firstName, lastName } = await getUserDetails(recall.assessedByUserId, res.locals.user.token)
+        res.locals.assessedByUserName = `${firstName} ${lastName}`
+      } catch (err) {
+        // What do we do if getUserDetails fails?
+        res.locals.assessedByUserName = recall.assessedByUserId
```

Comment:
Maybe set to 'Not available'? The user ID won't mean much to the viewer

Reply:
It was on the AC of the story:
`In the situation where we don’t have the name for a user, we’d display their user ID.`
Can ask Ollie / Jason and confirm?

Reply:
ah fair enough